pkgs/top-level/all-packages.nix

```diff
@@ -13603,6 +13603,13 @@ with pkgs;
       ];
   };
 
+  linux_hardkernel_4_14 = callPackages ../os-specific/linux/kernel/linux-hardkernel-4.14.nix {
```

Comment:
Another thing I just noticed; do you want callPackage**s** here, or `callPackage` (no `s`) like the other kernels?

Reply:
Ugh, you're totally right. Thanks! Fixed.